batch_job/src/work_queue_factory.c

```diff
@@ -381,21 +381,19 @@ static int submit_worker( struct batch_queue *queue )
 		files = newfiles;
 	}
 	
-	if(os){
+	if(runos_os){
 		char* vc3_cmd = string_format("./vc3-builder --require cctools-statics -- %s",cmd);
-		char* temp = string_format("python /afs/crc.nd.edu/group/ccl/software/runos/runos.py %s %s",os,vc3_cmd);
+		char* temp = string_format("python /afs/crc.nd.edu/group/ccl/software/runos/runos.py %s %s",runos_os,vc3_cmd);
```

Comment:
make a #define CCTOOLS_RUNOS_PATH /afs/... at the top of this file, and then use "python " CCTOOLS_RUNOS_PATH " %s %s", etc